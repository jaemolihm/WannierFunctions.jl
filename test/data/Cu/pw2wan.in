
&inputpp
   prefix   = 'Cu'
   outdir   = './temp'
   seedname = 'Cu'
/

