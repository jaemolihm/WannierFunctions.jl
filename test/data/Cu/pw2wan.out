
     Program PW2WANNIER v.7.0 starts on  4Jul2022 at 21:53: 3 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      16 processor cores
     Number of MPI processes:                16
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      16
     28984 MiB available memory on the printing compute node when the environment starts
 
 
  Reading nscf_save data

     Reading xml data from directory:

     ./temp/Cu.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          27      27      9                  388      388      76
     Max          28      28     10                  393      393      79
     Sum         439     439    151                 6231     6231    1243
 
     Using Pencil Decomposition
 

     Check: negative core charge=   -0.000005
     Reading collected, re-writing distributed wavefunctions
 
  Spin CASE ( default = unpolarized )
 
  Wannier mode is: standalone     
 
  -----------------
  *** Reading nnkp 
  -----------------
 
  Checking info from wannier.nnkp file
 
  - Real lattice is ok
  - Reciprocal lattice is ok
  - K-points are ok
  - Number of wannier functions is ok (  7)
  - All guiding functions are given 
 
 Projections:
    0.000000    0.000000    0.000000  2  1  1    1.000000
    0.000000    0.000000    0.000000  2  2  1    1.000000
    0.000000    0.000000    0.000000  2  3  1    1.000000
    0.000000    0.000000    0.000000  2  4  1    1.000000
    0.000000    0.000000    0.000000  2  5  1    1.000000
   -0.250000    0.250000    0.250000  0  1  1    1.000000
    0.250000   -0.250000   -0.250000  0  1  1    1.000000
 
  Reading data about k-point neighbours 
 
  All neighbours are found 
 
  Opening pp-files 
 
 
  --------------------------
  *** Compute  A projections
  --------------------------
 
 Number of local k points =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27
  AMN calculated
 
  ---------------
  *** Compute  M 
  ---------------
 
 Number of local k points =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27
  MMN calculated
 
  -----------------------------------
  *** Orbital terms are not computed 
  -----------------------------------
 
  -----------------------------------
  *** SHC terms are not computed 
  -----------------------------------
 
  ----------------
  *** Write bands 
  ----------------
 
 
  -----------------------------
  *** Plot info is not printed 
  -----------------------------
 
  -----------------------------
  *** Parity info is not printed 
  -----------------------------
 
  ------------
  *** Stop pp 
  ------------
 
 
     init_pw2wan  :      0.28s CPU      1.70s WALL (       1 calls)
     compute_amn  :      0.50s CPU      0.55s WALL (       1 calls)
     compute_mmn  :      0.21s CPU      0.33s WALL (       1 calls)

     Internal routines:
     compute_u_kb :      0.18s CPU      0.19s WALL (     216 calls)
 
     PW2WANNIER   :      1.00s CPU      2.95s WALL

 
   This run was terminated on:  21:53: 6   4Jul2022            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
 
